### DISCUSSION TITLE
Font Size

### DISCUSSION THREAD

**Discussion by abbottcam** [totaldebug/atomic-calendar-revive#1472] [Showroom]
Hello

Would it be possible to increase the font size of the diary entries.

thanks

**Comment by BobBurgess** [comment]
I was able to increase the font size by adding the following code to the configuration, you need to install card_mod:

card_mod:
  style: |
    ha-card {
      font-size: 28px; !important;
    }
    
 However, the problem with this is that it appears a bit cramped. See below:
![Screenshot 2024-04-11 083830](https://github.com/totaldebug/atomic-calendar-revive/assets/150232608/4b33522a-83b6-49c1-b6dd-c2ba829070ee)

I'm still trying to figure out the correct code to add to help this situation. I've tried 

.daywrap {
    padding: 40px 0px 40px; !important;
    }

but that doesn't seem to work. It should be possible to get this working with card mod. Just need to figure out the correct changes to make.

**Reply by jackm96** [reply]
Did you work this out? I am trying to do the same!

**Reply by jackm96** [reply]
I worked it out. Add the below. Change the value to what suits you. 

```
    ha-card {
    line-height: 150% !important
    }
```

**Reply by BobBurgess** [reply]
Thankyou jackm96! This works a treat!
